mscore/palette/palettelistview.cpp

```diff
@@ -105,4 +108,38 @@ void PaletteListView::keyPressEvent(QKeyEvent* event)
                         QListView::keyPressEvent(event);
             }
       }
-}
+
+//---------------------------------------------------------
+//   PaletteListView::setupStyle
+//---------------------------------------------------------
+
+void PaletteListView::setupStyle()
+      {
+      QPalette pal = palette(); // color palette
+      QColor c;
+      if (preferences.getBool(PREF_UI_CANVAS_FG_USECOLOR)
+         && preferences.getBool(PREF_UI_CANVAS_FG_USECOLOR_IN_PALETTES))
+            c = preferences.getColor(PREF_UI_CANVAS_FG_COLOR);
+      else
+            c = QColor("#f9f9f9");
```

Comment:
I think for this color you need to make a constant and use it.

Reply:
I wonder why not just use `c = preferences.getColor(PREF_UI_CANVAS_FG_COLOR);`` in any case?

Reply:
@igorkorsukov, where would this constant go? What would it achieve given that it is only used here?

@Jojo-Schmitz, surely if the user turns off `PREF_UI_CANVAS_FG_USECOLOR_IN_PALETTES` then we can't just go ahead and use it anyway; we need to provide a sensible fallback.

In any case my code is identical to what is already done in [PaletteWidget::setupStyle](https://github.com/musescore/MuseScore/blob/414808fdb22dee33c5aa77449b4335bbc9b9f243/mscore/palette/palettewidget.cpp#L130-L133), which was written by @dmitrio95.

Reply:
I'm in the process of finishing up a PR that fixes a bunch of color-related bugs. Part of that is removing all hard-coded colors wherever possible. I've changed `PaletteWidget::setupStyle()` so that it calls `preferences.defaultValue(PREF_UI_CANVAS_FG_COLOR).toString()` instead of hard-coding the string `"#f9f9f9"`.

Would you mind making the same change here so that I don't have to fix it later?

Reply:
@Spire42, sure, no problem.